### ISSUE TITLE
Purchased music packs in Quest Beat Saber not available after latest BMBF install

### ISSUE THREAD

**Issue by esl1890** [kihecido/BMBF#9]
I have downloaded all my purchased packs, did a level in Solo and installed the latest BMBF and all the music packs show not purchased. Tried the process 3 times.

**Comment by slashy** [comment]
Same issue here. I bought all the packs before but after I updated BMBF they're gone. 

This needs a fix.

**Comment by Arakon** [comment]
Mine show up fine and I was able to download all of them earlier today. Played several a few mins ago, in fact.

**Comment by slashy** [comment]
> Mine show up fine and I was able to download all of them earlier today. Played several a few mins ago, in fact.

Can you confirm that you are on the latest version of BMBF (1.5)? Because at least a few hours ago today the windows installer still installed an earlier version (1.4.5) in which said songpacks still work (tho custom songs show up but won't work).

**Comment by divine332** [comment]
im on the newest everything and have done everything correct, yet customsongs dont work atm
(all locked)
When is fix coming?

**Comment by Arakon** [comment]
I'm on BMBF 1.5.0, installed via APKs only, I didn't bother with the PC installer.
Everything working flawlessly, songs, playlists, sabers all there.

**Comment by esl1890** [comment]
Well, the strange things in my case is the music packs suddenly allowed me to download them again. I had the Quest connected to my Mac and was updating some custom songs through the browser. I then went downstairs to play a few songs and went to music packs and it allowed me to download them again. No idea what happened.

**Comment by slashy** [comment]
> 
> 
> Well, the strange thing in my case is the music packs suddenly allowed me to download them again. I had the Quest connected to my Mac and was updating some custom songs through the browser. I then went downstairs to play a few songs and went to music packs and it allowed me to download them again. No idea what happened.

You're right! I let my quest sitting yesterday and now everything works again. Only thing I know has changed is that the quest updated to V12 with hand tracking.

**Comment by Futuremappermydud** [comment]
Most of the time you download the songs before you install bmbf
